operations/node_management/Add_Remove_Replace_NCNs/Redeploy_Services.md

```diff
@@ -0,0 +1,270 @@
+## Redeploy Services Impacted by Adding or Permanently Removing Storage Nodes
+
+This procedure redeploys S3 and SYSMGMT_HEALTH services to add or remove storage node endpoints.
+
+**This procedure can be skipped if a worker or master node have been added.** 
+
+- Proceed to the next step to [Validate NCN](Validate_NCN.md) or return to the main [Add, Remove, Replace or Move NCNs](../Add_Remove_Replace_NCNs.md) page.
+
+**This procedure can be skipped if a worker or master node have been removed.**
+
+- Proceed to the next step to [Validate Health](Validate_Health.md) or return to the main [Add, Remove, Replace or Move NCNs](../Add_Remove_Replace_NCNs.md) page.
+
+Otherwise, proceed with the following if a storage node has been added or removed.
+
+### Prerequisite
+
+The docs-csm RPM has been installed on the NCN. Verify that the following file exists:
+
+```bash
+   ncn-m# ls /usr/share/docs/csm/scripts/operations/node_management/Add_Remove_Replace_NCNs/update_customizations.sh
+```
+
+### Procedure
+
+### 1. Update the `nmn_ncn_storage` list to include the IPs for any added or removed storage nodes.
+
+#### 1.1 Acquire site-init.
+
+Before redeploying the desired charts, update the `customizations.yaml` file in the `site-init` secret in the `loftsman` namespace.
+
+1. If the `site-init` repository is available as a remote repository [as described here](../../../install/prepare_site_init.md#push-to-a-remote-repository), then clone it to ncn-m001. Otherwise, ensure that the `site-init` repository is available on ncn-m001.
```

Comment:
```suggestion
1. If the `site-init` repository is available as a remote repository [as described here](../../../install/prepare_site_init.md#push-to-a-remote-repository), then clone it to `ncn-m001`. Otherwise, ensure that the `site-init` repository is available on `ncn-m001`.
```